funnel/utils/markdown/base.py

```diff
@@ -89,7 +88,7 @@ class MarkdownConfig:
         'default', 'zero', 'commonmark', 'js-default', 'gfm-like'
     ] = 'commonmark'
     #: Updated options against the preset
-    options_update: Optional[Dict[OptionStrings, bool]] = None
+    options_update: Optional[Mapping] = None
```

Comment:
This should be `Mapping[type, type]` or `dict` if not specifying types.

Reply:
`Mapping` is equivalent to `Mapping[str, Any]`